mathics/core/parser/tokeniser.py

```diff
@@ -206,6 +206,9 @@
     ('Therefore', r' \u2234 '),
     ('Because', r' \u2235 '),
     ('Backslash', r' \u2216 '),
+
+    ('Information', r'\?\?'),
+    ('Definition', r'\?'),
```

Comment:
My point is that `PatternTest` and `Information` need to share the ` \? ` token.